config/prow/plugins.yaml

```diff
@@ -193,7 +193,7 @@ config_updater:
       clusters:
         gardener-prow-trusted:
         - prow
-    config/prow/cluster/webserver-static-files/*:
+    config/prow/webserver-static-files/*:
```

Comment:
This is the only deployment-related thing left in the `config/prow` directory.
We could remove the old deployment of the `webserver-static-files` folder via the `config_updater` plugin and deploy it with flux as well :)